design/resources.go

```diff
@@ -158,3 +158,179 @@ var _ = Resource("login", func() {
 		Response(Unauthorized)
 	})
 })
+
+var _ = Resource("tracker", func() {
+	BasePath("/trackers")
+
+	Action("list", func() {
+		Routing(
+			GET(""),
+		)
+		Description("List all tracker configurations.")
+		Params(func() {
+			Param("filter", String, "a query language expression restricting the set of found items")
+			Param("page", String, "Paging in the format <start>,<limit>")
+		})
+		Response(OK, func() {
+			Media(CollectionOf(Tracker))
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+
+	Action("show", func() {
+		Routing(
+			GET("/:id"),
+		)
+		Description("Retrieve tracker configuration for the given id.")
+		Params(func() {
+			Param("id", String, "id")
+		})
+		Response(OK, func() {
+			Media(Tracker)
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+
+	Action("create", func() {
+		Routing(
+			POST(""),
+		)
+		Description("Add new tracker configuration.")
+		Payload(CreateTrackerAlternatePayload)
+		Response(Created, "/trackers/.*", func() {
+			Media(Tracker)
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+	Action("delete", func() {
+		Routing(
+			DELETE("/:id"),
+		)
+		Description("Delete tracker configuration.")
+		Params(func() {
+			Param("id", String, "id")
+		})
+		Response(OK)
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+	Action("update", func() {
+		Routing(
+			PUT("/:id"),
+		)
+		Description("Update tracker configuration.")
+		Payload(UpdateTrackerAlternatePayload)
+		Response(OK, func() {
+			Media(Tracker)
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+
+})
+
+var _ = Resource("trackerquery", func() {
+	BasePath("/trackerqueries")
+
+	Action("list", func() {
+		Routing(
+			GET(""),
+		)
+		Description("List all tracker queries.")
+		Params(func() {
+			Param("filter", String, "a query language expression restricting the set of found items")
+			Param("page", String, "Paging in the format <start>,<limit>")
+		})
+		Response(OK, func() {
+			Media(CollectionOf(TrackerQuery))
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+
+	Action("show", func() {
+		Routing(
+			GET("/:id"),
+		)
+		Description("Retrieve tracker configuration for the given id.")
+		Params(func() {
+			Param("id", String, "id")
+		})
+		Response(OK, func() {
+			Media(TrackerQuery)
+		})
+		Response(BadRequest, func() {
+			Media(ErrorMedia)
+		})
+		Response(InternalServerError)
+		Response(NotFound)
+	})
+
+	Action("create", func() {
+		Routing(
+			POST(""),
+		)
+		Description("Add new tracker query.")
+		Payload(CreateTrackerQueryAlternatePayload)
```

Comment:
The Link feature is to aggregate the results, so for now we have removed linking.